### PR TITLE
micronaut-oci-atp fallbacks

### PR DESCRIPTION
**Pull request by pgressa**

### CODE REVIEW

File: developer-library/micronaut-oci-atp/deploy/deploy.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ In this lab you will:
 
 ### Deploy (From Local To VM)
 
+Note that steps 4 and 5 are not necessary if you deployed Helidon native image in previous Lab to VM.
```
Comment (graemerocher) [inline developer-library/micronaut-oci-atp/deploy/deploy.md:21]: Should be `Note that steps 4 and 5 are not necessary if you deployed the Helidon native image in the previous Lab to the VM.`

File: developer-library/micronaut-oci-atp/deploy/deploy.md [hunk 2]
```diff
@@ -80,6 +82,45 @@ In this lab you will:
     </copy>
     ```
 
+### _Alternative: Build and run on VM only_
+If you ran into trouble building or deploying microservices in previous steps, you may follow next instructions to build and deploy microservices directly on VM:
```
Comment (graemerocher) [inline developer-library/micronaut-oci-atp/deploy/deploy.md:86]: Should be `If you ran into trouble building or deploying the Microservice applications presented in previous steps, you may use the following instructions to build and deploy them directly to the OCI VM:`

File: developer-library/micronaut-oci-atp/helidon/helidon.md [hunk 3]
```diff
@@ -62,6 +62,31 @@ We can verify this application works by exercising the following endpoints:
 - `curl -H 'Accept: application/json' http://localhost:8081/metrics` - returns metrics (MicroProfile metrics format)
 - `curl -i http://localhost:8081/openapi` - returns OpenAPI yaml service description (MicroProfile OpenAPI specification)
 
+#### _Alternative: Run the Helidon MP microservice on deployed VM_
+As an alternative, we may run the Helidon MP microservice on VM that was deployed as part of the Terraform plan.
```
Comment (graemerocher) [inline developer-library/micronaut-oci-atp/helidon/helidon.md:66]: Should be `As an alternative, you may run the Helidon MP microservice on the VM that was deployed as part of the Terraform plan.`

File: developer-library/micronaut-oci-atp/helidon/helidon.md [hunk 4]
```diff
@@ -62,6 +62,31 @@ We can verify this application works by exercising the following endpoints:
...
+
+    ```
+    <copy>
+    curl https://objectstorage.us-phoenix-1.oraclecloud.com/n/toddrsharp/b/micronaut-lab-assets/o/helidon-mp-service -o /app/helidon-mp-service
+    chmod +x /app/helidon-mp-service
+    ./app/helidon-mp-service
+    </copy>
+    ```
+
+We can verify this application works by exercising the following endpoints:
```
Comment (graemerocher) [inline developer-library/micronaut-oci-atp/helidon/helidon.md:86]: Should be `You can now verify the application works by exercising the following endpoints:`

File: developer-library/micronaut-oci-atp/helidon/helidon.md [hunk 5]
```diff
@@ -181,6 +206,8 @@ micronaut:
 </copy>
 ```
 
+Note that if you deployed Helidon MP microservice in **STEP 1** to VM use as `pet-health` url: `http://[VM IP Address]:8081`
```
Comment (graemerocher) [inline developer-library/micronaut-oci-atp/helidon/helidon.md:209]: Should be `Note that if you deployed the Helidon MP microservice in **STEP 1** directly to the OCI VM  use the following URL for the pet-health configuration described below.`

**Review by graemerocher — CHANGES_REQUESTED** [review]

**Comment by graemerocher** [comment]
@tmcginn @kaymalcolm approved, please merge

**Review by kaymalcolm — APPROVED** [review]

**Pull request merged**